### DISCUSSION TITLE
Have a plan to suppoer pure react project without nextjs or vercel project?

### DISCUSSION THREAD

**Discussion by littledian** [vercel/turborepo#6429] [Help] [answered]
I found that it works around vercel project, is is have a plan to support pure web app with pure react、html and css？If so,  any idea about when to support ?

**Comment by anthonyshew** [comment, marked as answer]
Turborepo works with the conventions of Node.js monorepos but doesn't have an opinion about your underlying framework decisions. What you're proposing is supported!
